queries.sql

```diff
@@ -18,3 +18,85 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+
+BEGIN TRANSACTION;
+
+
+SELECT * FROM animals;
+
+
+ROLLBACK;
+
+
+SELECT * FROM animals;
+
+
+BEGIN TRANSACTION;
+
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+
+Update animals SET species = 'pokemon' WHERE species IS NULL;
+
+
+COMMIT;
+
+
+SELECT * FROM animals;
+
+
+BEGIN TRANSACTION;
+
+
+DELETE FROM animals;
+
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+
+
+BEGIN TRANSACTION;
+
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+
+SAVEPOINT s1;
+
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+
+ROLLBACK TO s1;
+
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+
+COMMIT
+
+
+SELECT COUNT(*) FROM animals;
+
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+
+SELECT AVG(weight_kg) FROM animals;
+
+
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+
+
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+
+
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth 
+BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
```

Comment:
- Good job done so far, but please attach screenshots for the results of each of these queries as required by the project requirements, please fix this 🙏